webapp/locales/de.json

```diff
@@ -417,7 +416,40 @@
+  "termsAndConditions": {
+    "risk": {
+      "title": "Unfallgefahr",
+      "description": "Das ist eine Testversion! Alle Daten, Dein Profil und die Server können jederzeit komplett vernichtet, verloren, verbrannt und vielleicht auch in der Nähe von Alpha Centauri synchronisiert werden. Die Benutzung läuft auf eigene Gefahr. Mit kommerziellen Nebenwirkungen ist jedoch nicht zu rechnen."
+    },
+    "data-privacy": {
+      "title": "Du und deine Daten",
+      "description": "Bitte beachte, dass wir die Inhalte der Alphaversion zu Werbezwecken, Webpräsentationen usw. verwenden, aber wir glauben, dass das auch in Deinem Interesse ist. Am besten keinen Nachnamen eingeben und bei noch mehr Datensparsamkeit ein Profilfoto ohne Identität verwenden. Mehr in unserer <a href=\"/privacy\" target=\"_blank\">Datenschutzerklärung</a>"
+    },
+    "work-in-progress": {
+      "title": "Baustellen",
+      "description": "Das ist immer noch eine Testversion. Wenn etwas nicht funktioniert, blockiert, irritiert, falsch angezeigt, verbogen oder nicht anklickbar ist, bitten wir dies zu entschuldigen. Fehler, Käfer und Bugs bitte melden! <a href=\"mailto:support@human-connection.org\" target=\"_blank\">support@human-connection.org</a>"
+    },
+    "code-of-conduct": {
+      "title": "Verhaltenscodex",
+      "description": "Die Verhaltensregeln dienen als Leitsätze für den persönlichen Auftritt und den Umgang untereinander. Wer als Nutzer im Human Connection Netzwerk aktiv ist, Beiträge verfasst, kommentiert oder mit anderen Nutzern, auch außerhalb des Netzwerkes, Kontakt aufnimmt, erkennt diese Verhaltensregeln als verbindlich an:  <a href=\"https://alpha.human-connection.org/pages/code-of-conduct\" target=\"_blank\"> https://alpha.human-connection.org/pages/code-of-conduct </a>"
```

Comment:
We cannot link to a static page of the alpha here!
https://alpha.human-connection.org/pages/code-of-conduct

Once we turn off the alpha, this link will point into nirvana. @datenbrei do we have a code-of-conduct page on staging?